[Security Center] Add Incident response services tab instructions (#25669)
cloudflare/cloudflare-docs: src/content/docs/security-center/cloudforce-one/index.mdx

---

> import { Details, DashButton, LinkButton } from "~/components"

Reviewer: ```suggestion
import { Details, DashButton } from "~/components"
```

## Receive help for an incident

Cloudforce One allows you to receive help to improve your security posture or recover from a past incident. This allows you to easily report security incidents directly within the Cloudflare dashboard.

1. In the Cloudflare dashboard, go to the **Threat Intelligence** page.

   <DashButton url="/?to=/:account/security-center/threat-intelligence" />

2. Go to **Incident response services** then complete the following instructions:

- **Choose service**: 
   - Select among **Receive post-incident support**, **Request penetration tests**, **Conduct table-top exercises**, **Ask for general security advice**. 
   - Once you have chosen your desired service, select **Next**.
- **Provide request details**: 
   - Fill in the information needed based on the service you previously selected. Once you entered all the information, select **Next**.
   - Review and submit your request. Then, select **Submit**.
- Once you submit your request, Cloudforce One will reply to you as soon as possible.

## Request help for active attack

> If you want to stop active cyber attack, you can request assistance via the Clouflare dashboard.

Reviewer: ```suggestion
If you want to stop an active cyber attack, you can request assistance via the Cloudflare dashboard.
```

1. In the Cloudflare dashboard, go to the **Support** page and select your account.

> <LinkButton href="https://dash.cloudflare.com/?to=/:account/support" icon="external">Go to Support</LinkButton>

Reviewer: ```suggestion
<DashButton href="https://dash.cloudflare.com/?to=/:account/support" icon="external">Go to Support</DashButton>
```

Reviewer: I don't think LinkButton is used here? We should be using DashButton, right?

2. Select **Under attack**.
3. Under **Request help to stop active cyberattacks**, select **Request help**.
4. The dashboard will show you a pop-up where you will need to enter and confirm your phone number.

> 5. Once you have entered your phone number, select **Confirm number and request help**. Requesting help from the dashboard will page the an incident responder and you can expect a call-back as soon as possible. We advise you to wait for the call-back, and only use the phone-line in case you have not heard back from the team within 10 minutes.

Reviewer: ```suggestion
5. Once you have entered your phone number, select **Confirm number and request help**. Requesting help from the dashboard will page an incident responder and you can expect a call-back as soon as possible. We advise you to wait for the call-back, and only use the phone-line in case you have not heard back from the team within 10 minutes.
```
